pyproject.toml

```diff
@@ -56,7 +56,7 @@ exclude = [
 ]
 
 [tool.ruff.lint]
-select = ["I", "E", "W", "D", "B", "F", "N", "Q"]
+select = ["ARG", "I", "E", "W", "D", "B", "F", "N", "Q"]
```

Comment:
Checking for ARG001 would've caught the global variable use in function (since function arg wasn't used) that caused the bug in the notebook
https://docs.astral.sh/ruff/rules/unused-function-argument/#unused-function-argument-arg001